pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerBuilder.java

```diff
@@ -418,7 +418,19 @@
      *            the list of interceptors to intercept the producer created by this builder.
      * @return the producer builder instance
      */
-    ProducerBuilder<T> intercept(ProducerInterceptor<T> ... interceptors);
+    @Deprecated
+    ProducerBuilder<T> intercept(ProducerInterceptor<?> ... interceptors);
```

Comment:
Can you clarify the change here? Will that break binary compatibility?

Reply:
It's not required to change this signature, i'll fix it.